Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -402,6 +402,25 @@ public let CodeGenerators: [CodeGenerator] = [
         b.endIf()
     },
 
+    CodeGenerator("SwitchCaseGenerator", input: .anything) { b, cond in
+        b.beginSwitch(cond){
+            b.generateRecursive()
+        }
+
+        var caseStmts = [Variable]()
+        // Generate 0 or more switch cases
+        for _ in 0..<Int.random(in: 0...5) {
+            let caseStmt = b.randVar(ofType: b.type(of: cond)) ?? b.randVar()
```

Comment:
An alternative that I can think of is:
1. Add a check to prevent `cond` from being added to the case `caseStmts`. So that should prevent code like the one above from being generated.
2. If `caseStmts` is empty and we haven't generated any case statments, then add `cond` to the `caseStmts` and generate atleast one case statement.
WDYT?

Reply:
Right ok, sounds good. Yeah you're right, you'll have to deal with the (rare) scenario of `cond` being the only variable available in the program. Maybe you can do something like:
```
var caseValue: Variable
if let v = b.randVar(ofType: b.type(of: cond)), v != cond, !cases.contains(v) {
    caseValue = v
} else {
    caseValue = b.randVar()
}
cases.insert(caseValue)
```
That should achieve more or less the same I think. WDYT?
I still think it'd be ok to just `b.randVar()` for all cases, but I'm definitely fine either way!

Reply:
Let's keep it simple for now and just go with randVar we can always revisit it later if we need to introduce special handling for the rare case :)